source/management.rst

```diff
@@ -1,3 +1,14 @@
 =============================
  The Management of the PyCQA
 =============================
+
+- Creator: Ian Cordasco
+
+- Pep8 Administrator: Ian Lee
```

Comment:
Great, thanks!. I'll continue that conversation over on the issue in the tracker.